semmc-aarch32/src/SemMC/Architecture/ARM/ASL.hs

```diff
@@ -412,7 +401,7 @@ unsafeSymbol nm = case WI.userSymbol nm of
   Right s -> s
 
 dropUFPrefix :: String -> String
-dropUFPrefix nm = case List.stripPrefix "uf." nm of
+dropUFPrefix nm = case List.stripPrefix "uf_" nm of
```

Comment:
Can probably just delete this.

Reply:
Oh, good catch

Reply:
Okay, I did a warning cleanup pass and got rid of that